v2.2/enterprise-licensing.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Enterprise Licensing
```

Comment:
This page talks about how to get an enterprise licensing, right? Not sure Enterprise Features is the correct title.